macos/library/FLEKeyEventPlugin.m

```diff
@@ -0,0 +1,54 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "FLEViewController.h"
+#import "FLEViewController+Internal.h"
+#import "FLEKeyEventPlugin.h"
+
+static NSString *const kKeyEventChannel = @"flutter/keyevent";
+
+@implementation FLEKeyEventPlugin
+
+@synthesize controller = _controller;
+
+- (NSString *)channel {
+  return kKeyEventChannel;
+}
+
+- (void)handleMethodCall:(FLEMethodCall *)call result:(FLEMethodResult)result {
+  result(FLEMethodNotImplemented);
+}
+
+- (void)dispatchKeyEvent:(NSString *)type event:(NSEvent *)event {
```

Comment:
This needs a declaration comment (per style guide). It should also follow ObjC naming conventions that the word before the : names the param, so dispatchKeyEvent:(...)event ofType:(...)type